src/api/public/apidocs-new/paths/person_login_token.yaml

```diff
@@ -40,6 +40,87 @@ get:
           example:
             code: view_user_not_authorized
             summary: 'You are not authorized to view this User.'
+  tags:
+    - Person
+
+post:
+  summary: Create a new authentication token for a person.
+  description: |
+    Create a new authentication token for a person passed as parameter.
+
+    The token may be limited for a specific package. In this case the query parameters
+    'project' and 'package' should be provided.
```

Comment:
```suggestion
    `project` and `package` should be provided.
```

Reply:
I would use the code style for parameters in the description, it highlights it a bit